src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureSettings.css

```diff
@@ -39,13 +40,15 @@
   justify-content: flex-end;
   align-items: center;
   padding: 0;
+  font-size: 14px;
```

Comment:
We have a variable for this font size `--size-normal`. In this file `src/styles/variables.css`